src/utils.ts

```diff
@@ -111,7 +111,7 @@ const extractAccessToken = (authHeader: string): string | undefined => {
 const setTokeninfo = (req: Request): (data: Token) => void => {
   return (data: Token) => {
 
-    const tokeninfo = { ...data };
+    const tokeninfo = { ...data, ...{access_token: undefined} };
```

Comment:
Note: TS 4.x.x enforces the property to be deleted needs to be an optional - I guess this is less invasive than making the property optional in the type

Reply:
`const tokeninfo = { ...data, access_token: undefined };`

Reply:
```suggestion
    const tokeninfo = { 
       ...data, 
       access_token: undefined
    };
```

Reply:
Same idea, oneline or multiline doesn't matter for me, but destructuring shouldn't be needed.